{"name": "故障告警类型", "status": "passed", "attachments": [{"name": "log", "source": "da0f28ad-3a73-4fae-90ab-c5054588a9da-attachment.txt", "type": "text/plain"}], "start": 1691638746568, "stop": 1691638746603, "uuid": "32fcd0bf-e8a1-4828-98b1-d4260e9e869b", "historyId": "41246cfd2236c6b7ac7746b23ea9928d", "testCaseId": "3e54e5e903ce4b367ac549987cbede5f", "fullName": "opsPC.test_alarm#test_alarm_fault_type_001", "labels": [{"name": "feature", "value": "运维管理平台"}, {"name": "story", "value": "应急处置"}, {"name": "parentSuite", "value": "opsPC"}, {"name": "suite", "value": "test_alarm"}, {"name": "host", "value": "cale"}, {"name": "thread", "value": "3124-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "opsPC.test_alarm"}]}